README.md

```diff
@@ -71,6 +71,8 @@ Mutation testing is the practice of making better code by introducing bugs. As o
     * [Publications related to the Major mutation framework](http://mutation-testing.org/publ/)
 * JavaScript
   * [stryker-mutator/stryker](https://github.com/stryker-mutator/stryker)
+* Kotlin
+  * [mutant-kraken](https://github.com/JosueMolinaMorales/mutant-kraken)
```

Comment:
Should this be part of `Java/JVM` instead? IIRC pitest supports Kotlin too no?

Reply:
Pitest does support Kotlin because Kotlin compiles down to the JVM but there are not mutation operators specifically for Kotlin (ie. no operators that take advantage of kotlin only features)

That would be my argument for it being Kotlin only. Plus the tool is a source code level tool so it would not work on Java since it only parses Kotlin files.

Reply:
fair enough